cli/cmd/top.go

```diff
@@ -407,8 +410,14 @@ func getTrafficByResourceFromAPI(k8sAPI *k8s.KubernetesAPI, req *pb.TapByResourc
 	//       processEvents() && renderTable()
 	closing := make(chan struct{}, 1)
 	done := make(chan struct{})
+	scrollLeft := make(chan int)
```

Comment:
Since we never care about the actual data in these channels, they can be `chan struct{}` instead of `chan int`.  Just like the ones above.

Reply:
Alternatively, we could just use a single channel for `horizontalScroll` for both left and right events, putting in the x-offset for each event.

Reply:
@adleong This is my first time working with Go. I am not sure how to set an off-set for a channel. Could you please guide me on the same. Sorry for the inconvenience.

Reply:
No problem.  I just meant using a `chan int` which holds the amount to change the x-position by.  So whenever a left key press is detected, we would put `-5` into the channel and whenever a right key press is detected, we would put `5` into the channel (or whatever the scroll amount is).  This lets us use 1 channel instead of 2.